linux/kernel/building.md

```diff
@@ -103,10 +103,21 @@ You can either do this using VirtualBox (or VMWare) on Windows, or install it di
 
 To build the sources for cross-compilation, make sure you have the dependencies needed on your machine by executing:
 ```bash
-sudo apt install git bc bison flex libssl-dev make libc6-dev libncurses5-dev crossbuild-essential-armhf
+sudo apt install git bc bison flex libssl-dev make libc6-dev libncurses5-dev
 ```
+
 If you find you need other things, please submit a pull request to change the documentation.
 
+#### install the 32-bit toolchain for a 32-bit kernel
```

Comment:
We **do** want a capital `I` here, because it's the first word in the header :smile:

Reply:
My bad, miss understood your request. Fixed in f32ba83